xml/ay_bigfile.xml

```diff
@@ -1468,6 +1467,11 @@ exit 0
          nor <literal>reg_server</literal> are set, the system is registered with
          the &scc;.
         </para>
+        <para>
+         This URL can be used if needed while trying to determine the
+         self-update update repository. Check out the &deploy; to find further
+         information about this feature.
+        </para>
```

Comment:
The &smt; server is queried for an URL of the self-update update repository. So if <literal>self_update_url</literal> is not set, the &smt; server influences where the self-updates are downloaded from. Check out the &deploy; to find further
information about this feature.